manifest-el10-shared.yaml

```diff
@@ -1,8 +1,18 @@
 # Place here configuration that should happen on all el10* builds
 
+recommends: false
+
 include:
   - fedora-coreos-config/manifests/selinux-workaround.yaml
 
+# Manually include some packages now that disabled recommends
+packages:
+  - audit-rules
+  - redhat-release-eula
+packages-x86_64:
+  # Needs to be included at the bootc image level
+  - amd-ucode-firmware
```

Comment:
Let's just put these in `common.yaml`. If we've made a determination these packages should be in RHCOS they deserve to be there and not here.

Reply:
or maybe some of them don't exist in EL9 ?